SoftLayer/CLI/object_storage/list_endpoints.py

```diff
@@ -2,26 +2,83 @@
 # :license: MIT, see LICENSE for more details.
 
 import click
-
 import SoftLayer
 from SoftLayer.CLI import environment
 from SoftLayer.CLI import formatting
+from SoftLayer import utils
 
 
 @click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('identifier')
 @environment.pass_env
-def cli(env):
+def cli(env, identifier):
     """List object storage endpoints."""
 
     mgr = SoftLayer.ObjectStorageManager(env.client)
-    endpoints = mgr.list_endpoints()
+    endpoints = mgr.list_endpoints(identifier)
+
+    final_end_points = []
 
-    table = formatting.Table(['datacenter', 'public', 'private'])
+    table = formatting.Table(['Legacy', 'EndPoint Type', 'Public/Private', 'Location/Region', 'Url'])
```

Comment:
I'd prefer the order be as follows:

```suggestion
    table = formatting.Table(['Location/Region', 'Url', 'EndPoint Type', 'Public/Private', 'Legacy'])
```

Also, set the alignment for `location` and `url` to be left as that makes it read a little easier.

Reply:
Done